db/seeds.rb

```diff
@@ -80,9 +80,9 @@ def make_lang_skills
 def create_two_group_offers(group_offer_category)
   [
     FactoryGirl.create(:group_offer, necessary_volunteers: 2,
-      group_offer_category: group_offer_category,
+      group_offer_category: group_offer_category, creator: User.find_by(role: 'department_manager'),
```

Comment:
Why do group offer categories need creator? And how users are associated with categories? (Sorry if it is obvious)

Reply:
Is not a `group_offer_category` that is created here, it's `group_offer` and group_offer_category is passed to it

Reply:
True! Sorry!